documentation/docs/14-configuration.md

```diff
@@ -195,6 +196,10 @@ See [Prerendering](#ssr-and-javascript-prerender). An object containing zero or
 
 Enables or disables the client-side [router](#ssr-and-javascript-router) app-wide.
 
+### serverEntryPoint
+
+A file path to a custom entry point for the server. Passed to `vite.build.rollupOptions.input`. See [the Rollup docs](https://rollupjs.org/guide/en/#input) for more info.
+
```

Comment:
Not sure I understand this — why is this being documented as part of `config.kit`? isn't it purely an adapter concern?

Reply:
It's not really an adapter concern after this change because the code for it is now contained in `build/index.js`. Vite is now doing the bundling and it needs to know what the entry point is. Most of the time that will be an adapter-provided file. But the user may want to override it. Not sure if you have any other ideas for how we might specify this and allow it to be overriden?

Reply:
I've updated it to be an adapter option